backend_new/src/dtos/listings/listing-feature-create.dto.ts

```diff
@@ -1,8 +1,4 @@
 import { OmitType } from '@nestjs/swagger';
 import { ListingFeatures } from './listing-feature.dto';
 
-export class ListingFeaturesCreate extends OmitType(ListingFeatures, [
```

Comment:
since we aren't making changes to the ListingFeatures anymore I think we should just drop ListingFeaturesCreate and leverage ListingFeatures in those spot

Reply:
👍 I've deleted this dto